src/inc/corjit.h

```diff
@@ -121,34 +120,10 @@ enum CorJitFlag
 
 #endif // !defined(_TARGET_X86_) && !defined(_TARGET_AMD64_)
 
-#if defined(MDIL)
-
-    CORJIT_FLG_MDIL                = 0x00004000, // Generate MDIL code instead of machine code
-
-    // Safe to overlap with CORJIT_FLG_MAKEFINALCODE below. Not used by the JIT, used internally by NGen only.
-    CORJIT_FLG_MINIMAL_MDIL        = 0x00008000, // Generate MDIL code suitable for use to bind other assemblies.
-
-    // Safe to overlap with CORJIT_FLG_READYTORUN below. Not used by the JIT, used internally by NGen only.
-    CORJIT_FLG_NO_MDIL             = 0x00010000, // Generate an MDIL section but no code or CTL. Not used by the JIT, used internally by NGen only.
-
-#else // defined(MDIL)
-
     CORJIT_FLG_CFI_UNWIND          = 0x00004000, // Emit CFI unwind info
-
-#if defined(FEATURE_INTERPRETER)
-
     CORJIT_FLG_MAKEFINALCODE       = 0x00008000, // Use the final code generator, i.e., not the interpreter.
-
-#endif // defined(FEATURE_INTERPRETER)
-
-#if defined(FEATURE_READYTORUN_COMPILER)
-
     CORJIT_FLG_READYTORUN          = 0x00010000, // Use version-resilient code generation
 
-#endif // defined(FEATURE_READYTORUN_COMPILER)
```

Comment:
removing these FEATURE_INTERPRETER/FEATURE_readytoruncomiler ifdefs isn't related to MDIL, but in the interest of removing #ifdefs from the JIT-EE interface, I support it.